deployment/run-unit-tests.sh

```diff
@@ -65,7 +70,8 @@ cd $source_dir
 coverage_report_path=$source_dir/tests/coverage-reports/source.coverage.xml
 echo "coverage report path set to $coverage_report_path"
 
-pytest --cov --cov-report term-missing --cov-report term --cov-report "xml:$coverage_report_path" --cov-config=$source_dir/.coveragerc --ignore="tests/e2e"
+pytest $source_dir/tests --cov=$source_dir/glue/ --cov=$source_dir/helper/ --cov=$source_dir/amc_uploader/ --cov-report term-missing --cov-report term --cov-report "xml:$coverage_report_path" --ignore="tests/e2e" --ignore="api/tests/test_api_integration.py" --cov-fail-under=25 -vv
```

Comment:
Double check that you need to ignore api/tests/test_api_integration.py here. Seems strange you would need that since you're directing pytest to $source_dir/tests.

Reply:
resolved